app/views/artefacts/_migrated_app.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="callout callout-success">
+  <div class="callout-title">
+    Improved tagging
+  </div>
+  <div class="callout-body">
+    Tagging for this item
+    <%= link_to "has moved", Plek.find('content-tagger') + "/content/#{artefact.content_id}" %>.
+  </div>
+</div>
+<br/>
```

Comment:
Why the extra line breaks? You can use the class `add-bottom-margin` to give the callout some bottom margin.